src/DateUtils.ts

```diff
@@ -136,3 +136,29 @@ export function wantsDateSeparator(prevEventDate: Date, nextEventDate: Date): bo
     // Compare weekdays
     return prevEventDate.getDay() !== nextEventDate.getDay();
 }
+
+export function formatFullDateNoDay(date: Date) {
+    return (
+        date.getFullYear() +
+        "-" +
+        pad(date.getMonth() + 1) +
+        "-" +
+        pad(date.getDate()) +
+        _t(" at ") +
+        pad(date.getHours()) +
+        "." +
+        pad(date.getMinutes()) +
+        "." +
+        pad(date.getSeconds())
+    );
+}
+
+export function formatFullDateNoDayNoTime(date: Date) {
+    return (
+        date.getFullYear() +
+        "/" +
+        pad(date.getMonth() + 1) +
+        "/" +
+        pad(date.getDate())
+    );
```

Comment:
We could use a similar approach here for each language to specify their own common date format, e.g US:MM/DD/YYYY vs UK:DD/MM/YYYY

Reply:
Nad suggested this specific format. Should I change it?

Reply:
The relevant locale-aware JS API is https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/format but if the date isn't localised at all the default should probably be the ISO 8601 prefix dispensed from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString , e.g. YYYY-MM-DD (that is, first 10 characters of this API return)